packages/pocket-ic/src/common/rest.rs

```diff
@@ -20,6 +20,23 @@ pub struct AutoProgressConfig {
     pub artificial_delay_ms: Option<u64>,
 }
 
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize, JsonSchema, Default)]
+pub struct TickConfigs {
+    pub blockmakers_configs: Option<BlockMakerConfigs>,
```

Comment:
```suggestion
    pub blockmakers: Option<BlockMakerConfigs>,
```
all fields are configs as the struct name suggests